### PR TITLE
Combined supplier search box

### PR DESCRIPTION
**Pull request by katstevens**
Trello: https://trello.com/c/tZJCxC87/249-new-admin-search-box-for-suppliers

The new Data Controller role can search for suppliers in three ways: by name (prefix), DUNS number or Company Registration Number.

For now the CRN is just 'Companies House' number, not overseas registration numbers. There currently isn't a user need to search by overseas reg numbers.

Once this is merged to Preview, I'd like to do some thorough cross browser/device testing of the Design System elements before we deploy.

Relies on API and API client PRs to be merged first: (now done)
~https://github.com/alphagov/digitalmarketplace-api/pull/849~
~https://github.com/alphagov/digitalmarketplace-apiclient/pull/167~

### CODE REVIEW

File: requirements-app.txt [hunk 1]
```diff
@@ -9,4 +9,4 @@ lxml==4.2.5
 
 git+https://github.com/alphagov/digitalmarketplace-content-loader.git@5.0.0#egg=digitalmarketplace-content-loader==5.0.0
 git+https://github.com/alphagov/digitalmarketplace-utils.git@45.0.2#egg=digitalmarketplace-utils==45.0.2
-git+https://github.com/alphagov/digitalmarketplace-apiclient.git@19.7.0#egg=digitalmarketplace-apiclient==19.7.0
+git+https://github.com/alphagov/digitalmarketplace-apiclient.git@19.10.0#egg=digitalmarketplace-apiclient==19.10.0
```
Comment (lfdebrux) [inline requirements-app.txt]: Might be worth putting something in the commit message saying why the api client bump is needed

File: gulpfile.js [hunk 2]
```diff
@@ -239,6 +241,25 @@ gulp.task(
...
+  copyFactory(
+    "fonts from the GOVUK frontend assets",
+    govukFrontendFontsFolder,
+    assetsFolder + '/fonts'
+  )
+);
+
+
+gulp.task(
+  'copy:images',
```
[the remark this answers is not in the document]
Comment (katstevens) [reply gulpfile.js]: Ooops good spot! Will change

File: requirements-app.txt (hunk 1 above)
Comment (katstevens) [reply requirements-app.txt]: Now done 👍

File: gulpfile.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (katstevens) [reply gulpfile.js]: Added a commit to copy fonts straight to the static folder

**Pull request merged**